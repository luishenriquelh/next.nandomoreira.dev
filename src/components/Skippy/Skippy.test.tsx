import React from 'react';
import { render, cleanup } from '@testing-library/react';
import { Skippy } from '.';

describe('<Skippy />', () => {
  afterEach(cleanup);

  test('Render component', () => {
    const componentTree = render(<Skippy />);
    expect(componentTree).toMatchSnapshot();
  });
});
