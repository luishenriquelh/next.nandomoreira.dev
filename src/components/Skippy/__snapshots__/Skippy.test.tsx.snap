// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Skippy /> Render component 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": .c0 {
  border: 0;
  -webkit-clip: rect(0,0,0,0);
  clip: rect(0,0,0,0);
  height: 1px;
  margin: -1px;
  overflow: hidden;
  padding: 0;
  text-align: center;
  top: 40px;
  left: 40px;
  white-space: nowrap;
  width: 1px;
  z-index: 10001;
}

.c0 a {
  width: 100%;
  display: inline-block;
}

.c0:focus,
.c0:focus-within {
  -webkit-clip: auto;
  clip: auto;
  display: block;
  height: auto;
  margin: 0;
  overflow: visible;
  padding: 0.5rem;
  position: fixed;
  white-space: normal;
  width: auto;
}

<body>
    <div>
      <div
        class="c0"
      >
        <a
          href="#mainContent"
        >
          Ir para o conteúdo principal
        </a>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="Skippystyles__Skippy-becy8g-0 jPtRzB"
    >
      <a
        href="#mainContent"
      >
        Ir para o conteúdo principal
      </a>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
